ansible/roles/linux-perf/.kitchen.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+driver:
+  name: docker
+  use_sudo: false
+  cap_add:
+    - SYS_ADMIN
+
+platforms:
+  - name: ubuntu-14.04
+#  - name: ubuntu-16.04
```

Comment:
Can we test on ubuntu-16.04 instead of ubuntu-14.04? `linux-perf` role is executed only on ubuntu-16.04 machines right now (see https://github.com/nodejs/build/blob/master/ansible/playbooks/jenkins/worker/create.yml#L63-L66)

Reply:
Sure we can ;)